data/sql/esri.sql

```diff
@@ -12291,3 +12289,46 @@ INSERT INTO "grid_transformation" VALUES('ESRI','108453','AGD_1984_To_GDA_1994_N
+------------------
+-- ESRI grid names
+------------------
+INSERT INTO grid_alternatives(original_grid_name,
+                              proj_grid_name,
+                              old_proj_grid_name,
+                              proj_grid_format,
+                              proj_method,
+                              inverse_direction,
+                              package_name,
+                              url, direct_download, open_license, directory)
+VALUES
+('prvi','us_noaa_prvi.tif','prvi','GTiff','hgridshift',0,NULL,'https://cdn.proj.org/us_noaa_prvi.tif', 1, 1, NULL),
+('portugal/DLX_ETRS89_geo','pt_dgt_DLx_ETRS89_geo.tif','DLX_ETRS89_geo.gsb','GTiff','hgridshift',0,NULL,'https://cdn.proj.org/pt_dgt_DLx_ETRS89_geo.tif',1,1,NULL),
+('portugal/D73_ETRS89_geo','pt_dgt_D73_ETRS89_geo.tif','D73_ETRS89_geo.gsb','GTiff','hgridshift',0,NULL,'https://cdn.proj.org/pt_dgt_D73_ETRS89_geo.tif',1,1,NULL),
+('netherlands/rdtrans2008','','rdtrans2008.gsb','NTv2','hgridshift',0,NULL,'https://salsa.debian.org/debian-gis-team/proj-rdnap/raw/upstream/2008/rdtrans2008.gsb',1,0,NULL),
+('uk/OSTN15_NTv2','uk_os_OSTN15_NTv2_OSGBtoETRS.tif','OSTN15_NTv2_OSGBtoETRS.gsb','GTiff','hgridshift',1  -- reverse direction
+    ,NULL,'https://cdn.proj.org/uk_os_OSTN15_NTv2_OSGBtoETRS.tif',1,1,NULL),
+('canada/GS7783','ca_nrc_GS7783.tif','GS7783.GSB','GTiff','hgridshift',0,NULL,'https://cdn.proj.org/ca_nrc_GS7783.tif',1,1,NULL),
+('c1hpgn', 'us_noaa_c1hpgn.tif', 'c1hpgn.gsb', 'GTiff', 'hgridshift', 0, NULL, 'https://cdn.proj.org/us_noaa_c1hpgn.tif', 1, 1, NULL),
+('c2hpgn', 'us_noaa_c2hpgn.tif', 'c2hpgn.gsb', 'GTiff', 'hgridshift', 0, NULL, 'https://cdn.proj.org/us_noaa_c2hpgn.tif', 1, 1, NULL),
+('spain/100800401','es_cat_icgc_100800401.tif','100800401.gsb','GTiff','hgridshift',0,NULL,'https://cdn.proj.org/es_cat_icgc_100800401.tif',1,1,NULL)
+;
+-- 'france/RGNC1991_IGN72GrandeTerre' : we have a 3D geocentric corresponding one: no need for mapping
+-- 'france/RGNC1991_NEA74Noumea' : we have a 3D geocentric corresponding one: no need for mapping
+-- 'gc_nad83_harn_2007_conus_shifts': no mapping
+-- 'gc_nad83_harn_2007_alaska_shifts': no mapping
+-- 'gc_nad83_harn_2007_prvi_shifts': no mapping
+-- 'gc_nad83_2007_2011_conus_shifts': no mapping
+-- 'gc_nad83_2007_2011_alaska_shifts': no mapping
+-- 'gc_nad83_2007_2011_prvi_shifts': no mapping
+-- 'uk/osgb36_xrail84': no mapping
+-- 'spain/PENR2009': no mapping
+-- 'spain/BALR2009': no mapping
+-- 'japan/tky2jgd': no mapping
+-- 'spain/baleares': no mapping
+-- 'spain/peninsula': no mapping
+-- 'icegrid2004': no mapping
+-- 'ICEGRID93': no mapping
+-- 'imhpgn': no mapping
+-- 'ohdhihpgn': no mapping
+-- 'switzerland/ntv2-ch03p-etrs': no mapping
+-- 'australia/QLD_0900': no mapping
```

Comment:
From https://www.dnrme.qld.gov.au/__data/assets/pdf_file/0006/105765/gday-21-user-guide.pdf

    Note that the Queensland grid QLD_0900.gsb produces 
identical results to the National AGD84 grid for the equivalent coverage.

Reply:
Thanks